cli/src/main/java/denominator/cli/Denominator.java

```diff
@@ -309,6 +323,43 @@ public void run() {
       }
     }
 
+    private Function<Object, Object> attemptToTransformCredentials = new Function<Object, Object>() {
+      @Nullable
```

Comment:
this isn't adding much value.. I'd remove the nullable annotations here (and the import)